src/GlobalData.cpp

```diff
@@ -19,7 +20,7 @@ GlobalData& GlobalData::getInstance() {
 }
 
 GlobalData::GlobalData() {
-    QString urlBase = "http://s3.amazonaws.com/hifi-public";
+    QString urlBase = URL_BASE;
```

Comment:
The BASE_URL --> URL_BASE name change threw me for a loop until I saw it explicitly happen in **StackManagerVersion.h.in** below.